diagnostic.md

```diff
@@ -92,7 +123,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+The font-color will be yellow and the font-style will be sans-serif because the latest styling always wins.
```

Comment:
The font-family will be cursive due to the inline styling that always takes precedent.